docs/schedule/examples/fares-v2.md

```diff
@@ -216,7 +216,167 @@ The single ride fare product shown below has both `cash` and `tap-to-ride` fare
 <sup><a href="https://gtfs.calitp.org/production/CleanAirExpressFaresv2.zip" target="_blank">Download the Clean Air Express feed</a></sup>
 
 
+## Define price differences based on time of trip
 
+Some transit agencies differentiate fares depending on the time and/or day of the week, where fares can be associated with specific time periods such as a peak, off-peak and/or weekends. 
 
+Washington DC’s <a href="https://www.wmata.com/fares/basic.cfm" target="_blank">Metrorail fares</a> vary based on multiple factors, including day and time of trip. Variable time fares in GTFS can be defined using `timeframes.txt`, in which it is possible to designate specific time periods that then can be associated in `fare_leg_rules.txt` to assign the applicable fare product that corresponds to the time when the trip is made. 
```

Comment:
it could be slightly clearer that this is hypothetical (just so people don't mistakenly assume WMATA is in fact using time variable fares right now in their feed).

Reply:
The text has been updated to include both the "day and time of the trip" correction and the clarification that this is a fictional example